releasenotes/notes/fix-missing-pulse-lib-c370f5b9393d0df6.yaml

```
@@ -0,0 +1,11 @@
+---
+fixes:
+  - |
+    Fixed a bug that results in an error when a user tries to load .calibration
+    data of a gate in :class:`.Target` in a particular situation.
+    This occurs when the backend doesn't support pulse waveform payload, while
+    Qiskit doesn't have corresponding parametric pulse definition.
+    In this situation, Qiskit pulse object cannot be built, resulting in the failure in
+    parsing PulseQobj and the user encounters a meaningless error.
+    This error was suppressed, and the .calibration now returns None value instead,
+    which implies the calibration data is not properly supplied.
```

The part about supporting a waveform payload was throwing me off. I would say that this situation could occur and should be handled, but that it really shouldn't occur and is more of a bug in the backend service when it does. Here is some suggested wording:

```suggestion
    This occurs when the backend reports only partial calibration data, for
    example referencing a waveform pulse in a command definition but not
    including that waveform pulse in the pulse library. In this situation, the
    Qiskit pulse object cannot be built, resulting in a failure to build the pulse
    schedule for the calibration. Now when calibration data is incomplete
    the :class:`.Target` treats it as equivalent to no calibration being reported
    at all and does not raise an exception.
```

---

Yes, likely. Thanks for wording suggestion :)